src/core/services/covid-dashboard.service.spec.js

```diff
@@ -0,0 +1,22 @@
+import { CovidDashboardService } from "./covid-dashboard.service";
+
+const dashboard = new CovidDashboardService();
+
+describe("CovidDashboardService ", () => {
+    test("getFullInformationCountry shoud be return object", () => {
```

Comment:
`should return an object on getFullInformationCountry call`
the same for others
template is should do smth on action